consensus/parlia/parlia.go

```diff
@@ -884,6 +1310,22 @@ func (p *Parlia) Seal(chain consensus.ChainHeaderReader, block *types.Block, res
 			return
 		case <-time.After(delay):
 		}
+
+		err := p.assembleVoteAttestation(chain, header)
```

Comment:
There is a change in miner module recently that the miner will try mine blocks for multiple times during 3 seconds,  so not sure how this impact the logic here, please take care during testing.

Reply:
early broadcast not on master branch, have been revert
so need to take care during testing when importing early broadcast feature again